### ISSUE TITLE
Translation meta-issue

### ISSUE THREAD

**Issue by HugoGiraudel** [HugoGiraudel/sass-guidelines#96]
Let me open this issue so that I can talk to all of you kind folks who agreed to translate the guidelines at once. If any of you have a question about anything, please feel free to ask! 

Allow me to take this issue as an opportunity to repeat the process to start translating (as I said on every translation issue):

1. Fork the repository on GitHub;
1. Clone it on your personal workspace: `git clone git@github.com:USERNAME/sass-guidelines`;
1. Put yourself on the `gh-pages` branch: `git checkout gh-pages`;
1. Create a new branch called after your language: `git checkout -b <your_language's_abbreviation>` (e.g. `es` for Spanish).
1. Copy `index.md` in a folder named after your language: `cp index.md <your_language's_abbreviation>/index.md` (e.g. `es/index.md`).
1. Translate it. :)

Ping @varemenos, @kmargaritis, @PatrickKDK, @pierrechoffe, @ttomdewit, @klapec, @kaibrueckers, @magalhini, @soniar4i, @srescio, @pavelantolik.

**Comment by HugoGiraudel** [comment]
Ping @grawl, for the Russian translation.

**Comment by HugoGiraudel** [comment]
Note to translators: headings' `id` are auto-generated by Jekyll based on the headings; basically it's a slugify. Anyway, since you translate the titles, you also have to translate the anchors.

**Comment by HugoGiraudel** [comment]
Regarding the `donate.html` partial that is being included all over the place, here is how we will proceed. Please create `_includes/donate/<language>.html` (e.g. `_includes/donate/de.html`) and translate the content of current `_includes/donate.html` here.

I take care of everything else. ;)

**Comment by HugoGiraudel** [comment]
Regarding *Further readings* links, I think they should not be translated so it's bright clear they are in another language than the current one. Please, do leave them in English (also there is one in French if I'm right). :)

**Comment by HugoGiraudel** [comment]
Regarding Further readings links, I think they should not be translated so it's bright clear they are in another language than the current one. Please, do leave them in English (also there is one in French if I'm right). :)

Ping all: @varemenos, @kmargaritis, @PatrickKDK, @pierrechoffe, @ttomdewit, @klapec, @kaibrueckers, @magalhini, @soniar4i, @srescio, @pavelantolik, @grawl, @PavelDemyanenko.

**Comment by HugoGiraudel** [comment]
Regarding the `donate.html` partial that is being included all over the place, here is how we will proceed. 

Please create `_includes/donate/<language>.html` (e.g. `_includes/donate/de.html`) and translate the content of current `_includes/donate.html` here. 

Do not update the `{% include %}` tags in your `index.md`, they should still be `{% include donate.html %}`. 

I take care of everything else. ;)

Ping all: @varemenos, @kmargaritis, @PatrickKDK, @pierrechoffe, @ttomdewit, @klapec, @kaibrueckers, @magalhini, @soniar4i, @srescio, @pavelantolik, @grawl, @PavelDemyanenko.

**Comment by kmargaritis** [comment]
:smirk:

**Comment by Grawl** [comment]
Already working on mistakes with Russian translation, will PR my changes soon.

**Comment by HugoGiraudel** [comment]
In order to translate *Note &mdash;* [that is being added by CSS](https://github.com/HugoGiraudel/sass-guidelines/blob/gh-pages/_sass/components/_note.scss#L18) depending on the document language, I need you all to give me the translation for *Note* (either in this issue or the issue dealing with your translation). Thanks in advance! :)

Ping all: @varemenos, @kmargaritis, @PatrickKDK, @pierrechoffe, @ttomdewit, @klapec, @kaibrueckers, @magalhini, @soniar4i, @srescio, @pavelantolik, @grawl, @PavelDemyanenko.

**Comment by varemenos** [comment]
Great, thanks for the guidelines!

**Comment by HugoGiraudel** [comment]
Writing guidelines to translation guidelines. GUIDELINES-CEPTION!

**Comment by pierrechoffe** [comment]
Salut Hugo,

(1) Liens
Ok pour le lien vers les articles originaux en anglais (dommage, y’a de bonnes trad en français !! :))
Quid des liens vers wikipedia: idem, ou (lorsque c’est possible) lien vers wiki français?

(2) Traduction de Note
Vu que c’est le même mot (et si je comprends bien c’est la seule chose à traduire), ya rien de particulier à faire pour la trad fr ?

(3) Questions
Il y a des points sur lesquels j’ai des questions, donc je te propose d’en faire la liste une fois que j’aurai terminé, avant de te soumettre la trad, ok ?

@+, Pierre

**Comment by HugoGiraudel** [comment]
I just merged the new design changes that will allow your translations to be read. Because of this, you will have to synchronise your fork and rebase your branch folks.

First [configure a remote for your fork](https://help.github.com/articles/configuring-a-remote-for-a-fork/) if not done yet (this is a one time only operation):

```
$ git remote add upstream https://github.com/HugoGiraudel/sass-guidelines.git
```

Then [sync your fork](https://help.github.com/articles/syncing-a-fork/):

```
$ git fetch upstream 
$ git checkout gh-pages
$ git merge upstream/gh-pages
```

Last but not least, update your branch

```
$ git checkout <your_branch>
$ git stash
$ git fetch
$ git rebase origin/gh-pages
$ git stash pop
```

**Comment by HugoGiraudel** [comment]
Ping all: @varemenos, @kmargaritis, @PatrickKDK, @pierrechoffe, @ttomdewit, @klapec, @kaibrueckers, @altembo, @magalhini, @saravieira, @eduardoboucas, @soniar4i, @davidkpiano, @srescio, @pavelantolik, @grawl, @PavelDemyanenko.

I updated the [initial post](https://github.com/HugoGiraudel/sass-guidelines/issues/96#issue-54613655) with a lot of guidelines and tips. I'll update it again frequently.

**Comment by klapec** [comment]
Also, make sure you don't make the same dumb mistake like me and don't request your commits to be merged into `master`. `gh-pages` is where they should go.

**Comment by srescio** [comment]
I'm experiencing some issue with headings titles and anchors generated by Jekyll, it seems to have issues handling accented vowels, as result i get [Cosa è Sass] (#cosa-%C3%A8-sass) which doesn't work as proper anchor and doesn't make page scroll to the related element, if there's no fix/workaround I'll just replace them rephrasing the headlines that have this issue (only one so far), even though it would feel less "natural" to me

**Comment by HugoGiraudel** [comment]
Mmmh. I will have a look tomorrow.

**Comment by Grawl** [comment]
@srescio I have not worked on it but already see this problem and I think that we have add some HTML into our Markdown like this:

```markdown
<a href='#colors'>Цвета</a>

…

## <a name='colors'>Цвета</a>
```

But I don't know how it would work with existing Sass Guidelines frontend.

**Comment by HugoGiraudel** [comment]
Okay, so I just checked and "Cosa è Sass" leads to `cosa--sass`. So please don't refrain yourself using special characters. They are omitted in `id`s.

**Comment by Grawl** [comment]
@HugoGiraudel please note that only latin characters remains. Russian language don't have any latin characters so remains only words like “Sass” or other terms that we don't translate.

So if we have `Using Sass in your project` it will be translated like `Использование Sass в вашем проекте` and will be converted to something like `-sass---`.

**Comment by HugoGiraudel** [comment]
Mpf. That is an issue for Russian I suppose. Let me have a look, if I can find something.

**Comment by HugoGiraudel** [comment]
We might be onto something with this: http://kramdown.gettalong.org/options.html#option-transliterated-header-ids.

**Comment by HugoGiraudel** [comment]
Ping all: @varemenos, @kmargaritis, @PatrickKDK, @pierrechoffe, @ttomdewit, @bartveneman, @klapec, @kaibrueckers, @altembo, @magalhini, @saravieira, @eduardoboucas, @soniar4i, @davidkpiano, @srescio, @pavelantolik, @grawl, @PavelDemyanenko.

In order to avoid duplicating partials' HTML for each language, I slightly updated the way we work for the footer, the donate partial and the baseline. 

Instead of copy/pasting the HTML from `_includes/<partial>/en.html`, you will end up including `_includes/<partial>/template.html` in `_includes/<partial>/<language>.html`, passing it parameters. For instance:

```html
<!-- _includes/baseline/fr.html -->
{% include baseline/template.html content="Un guide arrêté pour écrire du code Sass propre, soutenable et évolutif." %}
```

I updated the initial post with an example for each of the three partials working like this.

**Comment by altembo** [comment]
@HugoGiraudel: Wouldn't it be helpful to add a Gitter-Room for further communication?

**Comment by HugoGiraudel** [comment]
Good idea. Done. If someone did not have an invitation, please ask.

https://gitter.im/HugoGiraudel/sass-guidelines

**Comment by HugoGiraudel** [comment]
Slight change in the content: https://github.com/HugoGiraudel/sass-guidelines/commit/50317adbcfc7b3ad966ee77214a8933e24f1ba82.

Ping all: @varemenos, @kmargaritis, @PatrickKDK, @pierrechoffe, @ttomdewit, @bartveneman, @klapec, @kaibrueckers, @altembo, @magalhini, @saravieira, @eduardoboucas, @couto, @soniar4i, @davidkpiano, @srescio, @pavelantolik, @grawl, @PavelDemyanenko.

**Comment by HugoGiraudel** [comment]
What would you folks think of having the content divided across multiple files? For instance, one file per major section? Would it help?

Ping all: @varemenos, @kmargaritis, @PatrickKDK, @pierrechoffe, @ttomdewit, @bartveneman, @klapec, @kaibrueckers, @altembo, @magalhini, @saravieira, @eduardoboucas, @couto, @soniar4i, @davidkpiano, @srescio, @pavelantolik, @grawl, @PavelDemyanenko.

**Comment by srescio** [comment]
I think so, as it would make detecting and managing changes easier

**Comment by HugoGiraudel** [comment]
I will think of something then.

**Comment by HugoGiraudel** [comment]
Main index would look like this:

```liquid
---
layout: default
language: en
---

{% include_relative /{{ page.language }}/01_author.md %}
{% include_relative /{{ page.language }}/02_contributing.md %}

{% include_relative /{{ page.language }}/03_sass.md %}
{% include_relative /{{ page.language }}/04_introduction.md %}
{% include_relative /{{ page.language }}/05_syntax.md %}

{% include donate.html %}

{% include_relative /{{ page.language }}/06_naming.md %}
{% include_relative /{{ page.language }}/07_comments.md %}
{% include_relative /{{ page.language }}/08_architecture.md %}
{% include_relative /{{ page.language }}/09_rwd.md %}

{% include donate.html %}

{% include_relative /{{ page.language }}/10_variables.md %}
{% include_relative /{{ page.language }}/11_extend.md %}
{% include_relative /{{ page.language }}/12_mixins.md %}
{% include_relative /{{ page.language }}/13_conditions.md %}

{% include donate.html %}

{% include_relative /{{ page.language }}/14_loops.md %}
{% include_relative /{{ page.language }}/15_errors.md %}
{% include_relative /{{ page.language }}/16_tools.md %}
{% include_relative /{{ page.language }}/17_tldr.md %}

{% include donate.html %}
```

I opened a PR to split the English version: https://github.com/HugoGiraudel/sass-guidelines/pull/111. Comments and review welcome. If it's fine with you, we'll implement it for all translations.

**Comment by kmargaritis** [comment]
I think it would be easier and will help a lot in the maintenance! :+1:  by me!

**Comment by HugoGiraudel** [comment]
I just merged the multi-files architecture into `gh-pages`. The English version now lives in `/en/`, divided across 18 files (one per major section). Your translation should reflect the exact same architecture in your `/<language>/` folder. 

The only exception with the English version is that you should have a `index.md` file containing nothing but this:

```liquid
---
layout: default
language: en
---

{% include chapters.html %}
```

Obviously replace `en` with your own language. That's it.

Ping all: @varemenos, @kmargaritis, @PatrickKDK, @pierrechoffe, @ttomdewit, @bartveneman, @klapec, @kaibrueckers, @altembo, @magalhini, @saravieira, @eduardoboucas, @couto, @soniar4i, @davidkpiano, @srescio, @pavelantolik, @grawl, @PavelDemyanenko.

**Comment by HugoGiraudel** [comment]
There are probably some words and expressions that should not be translated. Here is a list:

* TL;DR
* Too Long; Didn't Read
* front end / back end (noun)
* front-end / back-end (adjective)
* stylesheet(s)

**Comment by HugoGiraudel** [comment]
I added to the initial comment a list of words and expressions that should not be translated.

Ping all: @varemenos, @kmargaritis, @PatrickKDK, @pierrechoffe, @ttomdewit, @bartveneman, @klapec, @kaibrueckers, @altembo, @magalhini, @saravieira, @eduardoboucas, @couto, @soniar4i, @davidkpiano, @srescio, @pavelantolik, @grawl, @PavelDemyanenko.

@klapec Could you please make sure that's okay with the Polish version please? :)

**Comment by magalhini** [comment]
Ok, so I had already started (and commited) a few translations with the "old" way of doing it, ie, not with the split files. I'm having some trouble merging it all back together, so I reckon it should be safer for me to delete my "pt" branch remotely and branch of gh-pages again, fresh copy (and I'll manually paste the translations that I had done already).

Hopefully this should not cause any issues :)

**Comment by HugoGiraudel** [comment]
>  it should be safer for me to delete my "pt" branch remotely and branch of gh-pages again, fresh copy (and I'll manually paste the translations that I had done already).

Probably yes.

**Comment by klapec** [comment]
> @klapec Could you please make sure that's okay with the Polish version please? :)

Polish grammar is [tricky](http://en.wikipedia.org/wiki/Polish_grammar#Nouns), and I believe some of the abbreviations have to be [*declined*](http://en.wikipedia.org/wiki/Declension) (*CSSie, CSSu, HTMLu, etc.*)

I agree with your other points though. I wrote my translation with these in mind, except for few small things (like translating *too long; didn't read*). I'll fix it and do a PR.

**Comment by HugoGiraudel** [comment]
> I agree with your other points though. I wrote my translation with these in mind, except for few small things (like translating too long; didn't read). I'll fix it and do a PR.

You know the language better than I do. If you feel like it is okay the way you did it, then that's fine with me. The point is to make Sass Guidelines fluid and elegant in all languages. I trust you and any translator on this. :)

**Comment by HugoGiraudel** [comment]
Could I have a rough overview of the current status on each translation? There is no rush whatsoever, it is only to get an estimation of which language might be ready when. Thanks in advance folks. :)

Ping all: @varemenos, @kmargaritis, @PatrickKDK, @pierrechoffe, @ttomdewit, @bartveneman, @klapec, @kaibrueckers, @altembo, @magalhini, @saravieira, @eduardoboucas, @couto, @soniar4i, @davidkpiano, @srescio, @pavelantolik, @grawl, @PavelDemyanenko.

**Comment by altembo** [comment]
German: 3/18 Chapters translated

**Comment by soniar4i** [comment]
Spanish: 5/18 chapters translated

**Comment by pierrechoffe** [comment]
French: 50% translated

**Comment by Grawl** [comment]
At Russian we have successfully translated 5 full charapters and all other is translated as the base.

**Comment by srescio** [comment]
I've just committed the introduction, moving to _syntax.md

**Comment by magalhini** [comment]
My part is almost done (Parts of Syntax, Introduction, Author, Contributing) and already living in my fork. All I need now is to finish off some details, review it and make it consistent with the other portuguese peers who are translating their bits :)

**Comment by HugoGiraudel** [comment]
Alright. Glad to know things are moving forward. :)

**Comment by HugoGiraudel** [comment]
Ping all: @varemenos, @kmargaritis, @ttomdewit, @bartveneman, @kaibrueckers, @altembo, @magalhini, @saravieira, @eduardoboucas, @srescio, @pavelantolik.

Out of curiosity, what is the current progress on your respective translation?

**Comment by varemenos** [comment]
We've just started working on the Greek translation. There are like 2 .md files already translated and we will be pushing more soon. Hopefully, it will be ready in a week.

**Comment by HugoGiraudel** [comment]
@varemenos No rush my friend, it was only to have an ETA. :)

**Comment by srescio** [comment]
I've slowed down a little due to business, I think 2 weeks at most and it should be ready for first release

**Comment by HugoGiraudel** [comment]
I am kind of facing a dilemma here. I have [version 1.0.1](https://github.com/HugoGiraudel/sass-guidelines/pull/125) ready to be merged (reviews welcome, if you are interested in contributing to the English version of course). Problem is, once I've merged 1.0.1, all translations become (slightly) outdated.

What is best at this point: 
* waiting for all translations to hit 1.0.0, even if it means postponing 1.0.1 for several weeks? 
* merging 1.0.1, then updating existing translations to 1.0.1 and releasing new translations straight to 1.0.1 as well?

Opinions needed.

Ping all: @varemenos, @kmargaritis, @pierrechoffe, @ttomdewit, @bartveneman, @klapec, @matchabros, @kaibrueckers, @altembo, @magalhini, @saravieira, @eduardoboucas, @couto, @soniar4i, @davidkpiano, @srescio, @pavelantolik, @grawl, @PavelDemyanenko, @kimdhoe, @pinggod, @airen.

**Comment by soniar4i** [comment]
I think I'd choose the second option, even though all the translations become outdated, people could update them without having to wait till all the versions are released.

**Comment by klapec** [comment]
I'd say go for option #2.

**Comment by Grawl** [comment]
I think that second option with merging it today is a good approach, but additionally I suggest to add some banner with text like “this translation has outdated. see changelog to know what has changed in version 1.0.1”

**Comment by HugoGiraudel** [comment]
>  but additionally I suggest to add some banner with text for outdated translations. like “this translation has outdated. see changelog to know what has changed in version 1.0.1”. also will be cool to add a word “(outdated)” to language name in a list to let users know about update before they are switched to their language version.

This is already in place actually.

**Comment by Grawl** [comment]
It is in current deployed version or in 1.0.1?

I see only *“This version is exclusively maintained by contributors without the review of the main author, therefore might not be completely up-to-date.”*

I'm not about this. I'm about additional banner like about donation or mentioned above but red or something. Plus add a postfix to language select in sidebar.

**Comment by HugoGiraudel** [comment]
Is it currently deployed. When viewing a translation (a.k.a not the English version), a message says:

> This version is exclusively maintained by contributors without the review of the main author, therefore might not be completely up-to-date.

If the viewed version is lagging behind the English one (when comparing `version` key), the message says:

> This version is exclusively maintained by contributors without the review of the main author, therefore might not be completely up-to-date, especially since it is currently in version 1.0.0 while the [English version](http://sass-guidelin.es) is in version 1.0.1.

---

> Plus add a postfix to language select in sidebar. 

Now this is interesting. I like it.

**Comment by HugoGiraudel** [comment]
> Plus add a postfix to language select in sidebar.

Done. Live in current version.

**Comment by HugoGiraudel** [comment]
Please, when translating be sure to always translate `alt` attribute from images.

**Comment by Grawl** [comment]
Okay this is what I am talking about. Like it.

**Comment by kimdhoe** [comment]
The latter option seems better to me.

**Comment by HugoGiraudel** [comment]
Hey kind folks! How are things going? :)

Ping: @varemenos, @kmargaritis, @ttomdewit, @bartveneman, @kaibrueckers, @altembo, @magalhini, @saravieira, @eduardoboucas, @srescio, @pavelantolik.

**Comment by varemenos** [comment]
@kmargaritis and I have been buried in work lately so unfortunately things are progressing slowly.

**Comment by kaibrueckers** [comment]
Too much to do, too little time :(
But I reserved today for translating

**Comment by kmargaritis** [comment]
@varemenos and I on the same line today.. Translating today!

**Comment by HugoGiraudel** [comment]
Alright. I think I'll release 1.0.1 in two weeks or so. Probably around March 15th.

**Comment by HugoGiraudel** [comment]
Hey! Do any of you managed to move forward with a translation? :)

Ping: @varemenos, @kmargaritis, @ttomdewit, @bartveneman, @kaibrueckers, @altembo, @magalhini, @saravieira, @eduardoboucas, @srescio, @pavelantolik.

**Comment by varemenos** [comment]
Well, I'll try to avoid giving another excuse. We are slowly translating in our spare time and will eventually finish the translation. Hope that's not slowing down the progress of the project.

**Comment by HugoGiraudel** [comment]
> Well, I'll try to avoid giving another excuse. 

Haha, there is no problem whatsoever. It is only to have an estimation of what's going on with unfinished translation.

On my side, I'll release next version next week I suppose.

**Comment by srescio** [comment]
having translated just 5 sections then paused due to heavy business I think it's better for me to wait for new version to be released, so I can spend less time re-aligning and work directly on the new one

**Comment by HugoGiraudel** [comment]
Italian and Portuguese translations frozen until 1.1.0 gets out.

**Comment by HugoGiraudel** [comment]
Sass Guidelines EN 1.1.0 has been released just now.

**Comment by HugoGiraudel** [comment]
To those of you who already have translated version 1.0.0 successfully and want to add the additions from 1.1.0, the easiest way is described below.

## Adding a remote

Before starting though, make sure you have set a remote. It should be the case already but just in case it is not, let's double check.

```sh
git remote -v
```

If it only shows something like:

```sh
# origin  https://github.com/YOUR_USERNAME/sass-guidelines.git (fetch)
# origin  https://github.com/YOUR_USERNAME/sass-guidelines.git (push)
```

Then you need to add a remote, else just skip that part.

```sh
git remote add upstream https://github.com/HugoGiraudel/sass-guidelines.git
```

Now, just to make sure everything works:

```sh
git remote -v
# origin  https://github.com/YOUR_USERNAME/sass-guidelines.git (fetch)
# origin  https://github.com/YOUR_USERNAME/sass-guidelines.git (push)
# upstream  https://github.com/HugoGiraudel/sass-guidelines.git (fetch)
# upstream  https://github.com/HugoGiraudel/sass-guidelines.git (push)
```

## Syncing your fork

```sh
# Get changes from the original repository
git fetch upstream

# Go on the gh-pages branch
git checkout gh-pages

# Merge your local version with the one from the repository
git merge upstream/gh-pages
```

## Translating Sass Guidelines 1.1

```sh
# Create a branch named after your language (e.g. `ru-1.1.0`)
git checkout -b <your_language>-1.1.0
```

Then, I suppose the easiest way to make sure you forgot nothing is to replicate the commits from the 1.1 pull request one by one. Open [https://github.com/HugoGiraudel/sass-guidelines/pull/125](https://github.com/HugoGiraudel/sass-guidelines/pull/125) in a new tab, and apply commit one after the other.

For instance, the first one: https://github.com/HugoGiraudel/sass-guidelines/commit/4b400c50995bbda3d226725a871ccf98d552a28f. First, open the relevant files of your translation and apply changes. Then:

```sh
git add -A
git commit -m "Added extra further readings"
```

And then move on to the next one until you have applied them all. Once it is done, simply head into GitHub, and open a pull-request from your branch to the `gh-pages` one.

If you have any problem, please tell.

Ping @pierrechoffe, @klapec, @matchabros, @soniar4i, @davidkpiano, @grawl, @PavelDemyanenko, @airen, @pinggod, @kimdhoe.

**Comment by Grawl** [comment]
No problem, I know how to use SourceTree 😄

**Comment by pinggod** [comment]
@HugoGiraudel  what's the main changes?

**Comment by HugoGiraudel** [comment]
@pinggod: https://github.com/HugoGiraudel/sass-guidelines/pull/125.

**Comment by HugoGiraudel** [comment]
French version is now 1.1.0.

**Comment by HugoGiraudel** [comment]
First draft of contributing guidelines pushed to the repository: https://github.com/HugoGiraudel/sass-guidelines/blob/gh-pages/CONTRIBUTING.md.

**Comment by Grawl** [comment]
But where all the rules? For example, about what we should and what should not translate?

**Comment by HugoGiraudel** [comment]
Hence the word *draft*. :)

**Comment by Grawl** [comment]
Okay :) do you need our help? Maybe we should do some surveys?

**Comment by HugoGiraudel** [comment]
Massive changes:
* Table of contents is now automatically generated for each language; because of this, *further reading*-like headings are now level 6 (i.e. `###### Further reading`) so they are not part of the ToC
* Images now live in their own partial (`_includes/images/`); caption translations live in `_data/translations.yml`
* Code snippets now live in their own partial for all versions in 1.1.0 (`_includes/snippets/`); versions still in 1.0.0 keep code as part of chapters

**Comment by pinggod** [comment]
When zh version in 1.0.0, there are some error about ToC.
I pull request zh-1.1.0 just now, and wait for the result of changes.

**Comment by pinggod** [comment]
@HugoGiraudel 
When zh version in 1.0.0, there are some error about ToC.
I pull request zh-1.1.0 just now, and wait for the result of changes.

**Comment by HugoGiraudel** [comment]
Chinese version successfully ported to 1.1.0 thanks to @pinggod.

**Comment by HugoGiraudel** [comment]
The 7th translation of Sass Guidelines is released: this time in German by @morkro.

**Comment by HoBi** [comment]
Can you change the czech translator guy to me? :)

**Comment by HugoGiraudel** [comment]
Alright. :)

**Comment by HugoGiraudel** [comment]
Current progress on v1.2.0: https://github.com/HugoGiraudel/sass-guidelines/pull/223.

**Comment by HoBi** [comment]
:+1: 

I have to finish the Czech version finally. :trollface:

**Comment by HugoGiraudel** [comment]
That would be so great! :P

**Comment by HugoGiraudel** [comment]
I have updated the existing [`CONTRIBUTING.md`](https://github.com/HugoGiraudel/sass-guidelines/blob/gh-pages/CONTRIBUTING.md) file at the root of the repository. It now contains enough information about how to deal with Git, especially regarding the existing branching system.

I also realised my translating guidelines emitted in the original description of this issue sucked. I have been bad at this, and I sincerely apology. To solve this problem, I created a [`TRANSLATING.md`](https://github.com/HugoGiraudel/sass-guidelines/blob/gh-pages/TRANSLATING.md) file at the root of the repository, containing a lot of information about what should be translated, what shouldn't, and how to get started.

Then each language folder should have its own `README.md` file containing language-specific notes. I created a template for this here: https://github.com/HugoGiraudel/sass-guidelines/blob/gh-pages/en/README.md. **I would really appreciate if you all could copy this template in your own language folder, translate and fill it**. Feel free to submit a small pull-request when it is done. Thank you.&nbsp;:)

PS: our kind Portuguese-speaking friends already started this on their side: https://github.com/HugoGiraudel/sass-guidelines/blob/gh-pages/pt/README.md. 

Ping @pierrechoffe, @klapec, @matchabros, @morkro, @soniar4i, @davidkpiano, @grawl, @PavelDemyanenko, @airen, @pinggod, @kimdhoe, @AndersSchmidtHansen, @srescio, @HoBi, @ttomdewit, @bartveneman, @varemenos, @magalhini, @couto, @saravieira, @eduardoboucas.

**Comment by HugoGiraudel** [comment]
Forgot to ping you two: @wood1y, @jhonnymoreira. :)

**Comment by morkro** [comment]
Sure, will add the README.md later :+1:

**Comment by klapec** [comment]
I've added a README.md to the Polish translation. It's not yet 100% complete, I'd like to have someone else's input first. Should I just go ahead and submit a PR for it now? 
I've made a separate `pl-1.2.0` repo and I'll use it in the future when 1.2.0 is released.

**Comment by HugoGiraudel** [comment]
> Should I just go ahead and submit a PR for it now? 

Yes. Tag is `[RFR]`, and ask for someone to review it. :)

**Comment by morkro** [comment]
I got a question regarding the two lists of either translated or untranslated words. 

German is a bit of an exception here. We have this ability to turn any foreign word into a valid german one, by keeping the original word and just adding german stuff to it (like adding an article, postfixing it, ...), called _Germanising_. Where do I list them? It's not really translated in the classical way, but neither kept original. _Should I ignore them in general?_

I already asked some of my german colleagues and it's a difficult topic. It's still a technical term though, but made fit into proper german grammar. 
For instance the term **branch**. The german term would be _Zweig_ but in a technical context I would still refer to it as **[...] der Branch [...]**. It's english though, but the grammar changed to fit into proper german. It could now be used as a German term. :) 

So, to sum this up. Is it okay if I may add a third list for "eingedeutschte" (germanised) terms?

Sorry for my little German lession here.

**Comment by HugoGiraudel** [comment]
Maybe just explain this in the German README.md? :)

**Comment by morkro** [comment]
:+1: good point. I'll just add a little explanation regarding this then.

**Comment by HugoGiraudel** [comment]
About 24 hours in, already 3 translations (on 9) have their own README.md. Let me tell that I am very proud to have contributors as you folks. Thank you so much.

**Comment by HugoGiraudel** [comment]
## Sass Guidelines 1.2.0 is live! :sparkles: 

Dear friends, Sass Guidelines version 1.2.0 just got released with a lot of small additions (and a few bigger ones). A special thanks to @morkro for helping me proofreading everything.

## What it means for you

Depending on the state of your translation, it can mean:

* Updating it from 1.1.0 to 1.2.0 (in the best scenario)
* Updating it from 1.0.0 to 1.2.0 (without forgetting changes from 1.1.0)
* Translating it to 1.2.0 directly. It might be a bit tricky so what you can do I suppose is keep going until you're done with 1.1.0, and then apply the 1.2.0 changes. Your call.

In any case, please submit a pull-request when you are ready and we will work together to make sure the update is going seamlessly. :)

## Changelog

Here is an informal changelog extracted from the commit messages from the [1.2.0 pull-request](https://github.com/HugoGiraudel/sass-guidelines/pull/223):

- Updated the author section
- Added something about argument-less mixins
- Added something about extending the guidelines
- Added a mention of SitePoint Sass Reference
- Added a note about unitless zeros for durations
- Reworked the extend section
- Added a note about keyframes in 7-1 pattern
- Removed section about debugging a Sass map
- Clarified the postprocessors section
- Removed a lot of irrelevant further reads
- Changed the section about color formats
- Fixed an invalid advice regarding import order in the main Sass file
- Added something about the charset directive and single quotes
- Slightly improved the “Utils folder” sub-section
- Dramatically improved TL;DR section
- Slightly improved the Contributing section

Ping @pierrechoffe, @klapec, @matchabros, @morkro, @soniar4i, @davidkpiano, @grawl, @PavelDemyanenko, @airen, @pinggod, @kimdhoe, @AndersSchmidtHansen, @srescio, @HoBi, @ttomdewit, @bartveneman, @varemenos, @margaritis, @magalhini, @couto, @saravieira, @eduardoboucas, @wood1y, @jhonnymoreira, @shadeed.

**Comment by AndersSchmidtHansen** [comment]
Awesome! Great work!

I'll do my best to squeeze in time for updating the Danish translations. We're swamped with todos right now, so I apologize beforehand if I don't get to updating it in the near future. :)

**Comment by HugoGiraudel** [comment]
Important warning: Pay attention: the content of the commit https://github.com/HugoGiraudel/sass-guidelines/commit/4a4488864dd18888239290f8dc95b444e78b82e9 about reworking the `@extend` section has been changed as part of a later pull-request: https://github.com/HugoGiraudel/sass-guidelines/pull/259. Be sure to translate the content of that latter PR as well.

Ping @pierrechoffe, @klapec, @matchabros, @morkro, @soniar4i, @davidkpiano, @grawl, @PavelDemyanenko, @airen, @pinggod, @kimdhoe, @AndersSchmidtHansen, @srescio, @HoBi, @ttomdewit, @bartveneman, @varemenos, @margaritis, @magalhini, @couto, @saravieira, @eduardoboucas, @wood1y, @jhonnymoreira, @shadeed.

**Comment by magalhini** [comment]
Will have a closer look at this sometime this week, @HugoGiraudel ! Thanks for the updates :)

**Comment by HugoGiraudel** [comment]
Hey there,

It’s been a while since I haven’t checked up on you folks! It would be nice if you could give me the current status of your work (if any) for those of you who are moving from 1.1.0 (or even 1.0.0) to 1.2.0, and for those of you who are translating the guidelines in a new language for the first time.

Thank you for your understanding!

Ping @pierrechoffe, @klapec, @matchabros, @morkro, @soniar4i, @davidkpiano, @grawl, @PavelDemyanenko, @airen, @pinggod, @kimdhoe, @AndersSchmidtHansen, @srescio, @HoBi, @ttomdewit, @bartveneman, @varemenos, @margaritis, @magalhini, @couto, @saravieira, @eduardoboucas, @jhonnymoreira, @shadeed, @Mojtabarahimi8.

**Comment by varemenos** [comment]
The Greek translation is almost done. It will be based on v1.2.0.
We are in the process of proof reading it.

**Comment by HugoGiraudel** [comment]
Happy New Year dear contributors, and thank you for all the work you did on the guidelines in 2015. Needless to say I am super glad to have you onboard, and hope we can push this project even further in 2016. :)

Let me know if I can do anything to assist you!

**Comment by pierrechoffe** [comment]
À toi aussi mon cher Hugo, je te souhaite une très belle année et la réalisation de tes voeux les plus chers, tant personnels que professionnels. Continue de bien t’amuser chez ES, petit veinard !
Pierre

**Comment by Grawl** [comment]
I am happy to be onboard! Also I'm planning to finally update russian translation up to latest version because I'm sharing it too much because this guideline is crazy good to start with.

**Comment by HugoGiraudel** [comment]
Thank you so much @Grawl. I’m really looking forward to have the Russian translation up to date with the English one as it is the most needed and read translations of all. :)

**Comment by HugoGiraudel** [comment]
In case you are interested, here are the numbers for the period 01.09.2015 — 01.01.2016.

<img width="1161" alt="screen shot 2016-01-03 at 17 04 37" src="https://cloud.githubusercontent.com/assets/1889710/12079514/49b8aafa-b23c-11e5-9ba5-3c1f6da4b021.png">

**Comment by Grawl** [comment]
Wow! Russian translation is very popular. It's a great reason for me to update it :)

**Comment by varemenos** [comment]
Happy new year everyone! Damn those numbers are nice, didn't expect so much traffic for the translated versions. The greek translation was delayed once again but we are almost there like (99% done). I will be leaving for my 9 month mandatory military service in the 20th of this month so it will definitely be done by then.

**Comment by morkro** [comment]
Great traffic info, thanks :)
Would be great to see how many countries are visiting the english version though. For instance German is very low, is the rest browsing the English version or are Germans rare in general?

**Comment by HugoGiraudel** [comment]
Hey there!

The Greek translation of Sass-Guidelines has finally been finished and released this morning, thanks to great work from our friends in Greece. :tada:

I also took this as an opportunity to remove the third component of version numbers as it turns out to be completely obsolete. We will now refer to version as `1.0`, `1.1`, `1.2`, `1.3`, etc.

Speaking of which, there currently is a pull request in progress for version 1.3 if you are curious: https://github.com/HugoGiraudel/sass-guidelines/pull/292. 

Best! :)

**Comment by Grawl** [comment]
Moving from semver? Better using it, or it will too hard to maintain?

**Comment by HugoGiraudel** [comment]
The thing is we do not really need SemVer, and more than that: we cannot really use SemVer. As a reminder, for Sass-Guidelines the version number is used to keep translations in sync. If the document was only in English, it would not need to be versioned. 

In theory, the third component of a SemVer version number is used for bug fixes. In this project, it would mean typos. Typos are not versioned; they are directly merged in the main branch.

Which leaves us with version bumps used for additions, updates and removals of content, such as those which happened in 1.1, 1.2 and soon 1.3. These need to be bundled in versions so that contributors like you can actually see what happened on the English version and update their translation accordingly.

Hope it makes it clearer. :)

**Comment by HugoGiraudel** [comment]
Hey @soniar4i, @davidkpiano, @kimdhoe, @AndersSchmidtHansen, @magalhini, @couto, @saravieira, @eduardoboucas. Your translation is currently lagging behind the main one (1.1 against 1.2). Is there anything you need in order to update it? Can I do something to assist you? Or are you waiting for 1.3 to batch both versions into one go? Let me know. :)

**Comment by Grawl** [comment]
Okay, sounds reasonable.

**Comment by kimdhoe** [comment]
Korean 1.2 is almost there!

**Comment by HugoGiraudel** [comment]
@kimdhoe Great news! Looking forward to it. :)

**Comment by HugoGiraudel** [comment]
Hey there. I just merged PR #314 which introduces a relevant change for you contributors. The `_data` folder now contains one file per translation instead of a `translations.yml` and a `languages.yml` file. This will make diffs easier and generally makes more sense. :)

**Comment by margaritis** [comment]
It does make more sense! :+1:

**Comment by HugoGiraudel** [comment]
## Sass Guidelines 1.3 is live! :sparkles: 

Dear friends, Sass Guidelines (EN) version 1.3 just got released with a lot of small additions (and a few bigger ones).

## What it means for you as a translator

Depending on the state of your translation, it can mean:

* Updating it from 1.2 to 1.3 (in the best scenario)
* Updating it from 1.0 or 1.1 to 1.3 (without forgetting changes from [1.1](https://github.com/HugoGiraudel/sass-guidelines/blob/gh-pages/CHANGELOG.md#110-april-11th-2015) and [1.2](https://github.com/HugoGiraudel/sass-guidelines/blob/gh-pages/CHANGELOG.md#120-september-1st-2015))

In any case, please submit a pull-request (e.g. `[WIP] 1.3 French`) when you are ready and we will work together to make sure the update is going seamlessly. :)

## Changelog

[Here it is](https://github.com/HugoGiraudel/sass-guidelines/blob/gh-pages/CHANGELOG.md#130-march-1st-2016).

Ping @pierrechoffe, @klapec, @matchabros, @morkro, @soniar4i, @davidkpiano, @grawl, @PavelDemyanenko, @airen, @pinggod, @kimdhoe, @AndersSchmidtHansen, @srescio, @HoBi, @ttomdewit, @bartveneman, @varemenos, @margaritis, @magalhini, @couto, @saravieira, @eduardoboucas, @wood1y, @jhonnymoreira, @shadeed.

**Comment by morkro** [comment]
Awesome :tada: Will update the German version this week.

**Comment by HugoGiraudel** [comment]
Hey y’all beautiful people!

I would like to know if I can do anything to assist you in updating your translation to version 1.3. Do you feel confident upgrading it in the next few weeks? Else, shall I find you some support? Please, be sure to tell.

For instance, @nastiatikk is joining the Russian team to take over the current state of the Russian translation because @Grawl and @PavelDemyanenko do not have enough time. Welcome onboard!

Ping all: @pierrechoffe, @klapec, @matchabros, @morkro, @soniar4i, @davidkpiano, @grawl, @PavelDemyanenko, @airen, @pinggod, @kimdhoe, @AndersSchmidtHansen, @srescio, @HoBi, @ttomdewit, @bartveneman, @varemenos, @margaritis, @magalhini, @couto, @saravieira, @eduardoboucas, @jhonnymoreira, @shadeed, @nastiatikk.

**Comment by Grawl** [comment]
I found git too hard to update translations. Maybe some crowdsource translating service (like ones noticed in [this Quora page](https://www.quora.com/What-is-the-most-successful-translation-crowdsourcing-project-and-why)) will be better. Hunting for changes in commits for each file is terrible, but raw diff lurking is even harder.

**Comment by HugoGiraudel** [comment]
I’d argue the difficulty to update the Russian translation partially comes from the fact that it is lagging behind by 3 versions. However I will definitely concede that content translation process could be improved, and I’m well willing to investigate any possibility.

On my end, I already made sure that no translation file has any HTML for instance, to make sure diffs are not polluted with irrelevant changes.

**Comment by HugoGiraudel** [comment]
# Rethinking the translation system

Hey there. I have been thinking a lot about the current translation “system” recently, only to come to the conclusion that it’s really not great even if it seems to work. It has certainly gotten much better over the past year (some of you might remember the early days when the whole guidelines were in a single file). However, it’s still far from ideal…

That being said, we have 12 different languages live, and 5 are currently in the pipes. That’s huge for such a small open-source project. We have to find a way to make it work for everybody, and in the simplest way possible.

## What currently doesn’t work?

Translating the guidelines in a new language from scratch is actually not too hard. The main problem right now is updating a translation. Which is actually the most part of the translation work on Sass Guidelines. Let’s be honest, we don’t add a new language every day.

Their are two concurrent problems: it is hard to figure out what changed, and new versions are somewhat large (usually a dozen commits, involving a couple of hours of translation work). The current process for updating a translation is to check the [changelog](https://github.com/HugoGiraudel/sass-guidelines/blob/gh-pages/CHANGELOG.md), and to reproduce commits one by one. Not ideal. Translators do not really care about why something changed. They just want to know *what* changed and what they have to do on their translation to make it up-to-date.

This also means that translators currently need a bit of Git knowledge and that really sucks. Translation work has nothing to do with versioning, and someone with little to no technical background should be able to join and help. GitHub interface is able to bridge the gap a little, but I am not sure it’s enough given the current state of things. Which leads me to the next point.

On top of that, translators currently have to have some knowledge about the project and its technical setup. It is not mandatory to be able to run the project locally, but knowing where to find what and how it generally works appears like a prerequisite right now. Still, it should not. A translator should not have to know anything about the project setup. They should only be presented with content to be translated. Ideally, this content should be decoupled from any technical consideration.

Last but not least, as of today translators join to maintain a full version. While it is very good for consistency, it also asks a lot of time from them, sometimes more time than they can actually afford. I don’t want to force anybody do anything, especially given that it’s a non-profit open-source project. That’s also why I think the current system is broken. Someone should be able to come and translate a few sentences then leave, the same way translations work on Wikipedia for instance. Then, everybody could participate as much as they want, little by little. 

## What can we do?

I will be entirely honest with you and admit that I am not sure. I have little to no experience with internationalisation, and Sass Guidelines is pretty much a playground for me here. Needless to say I am incredibly happy and lucky to have you all onboard and willing to help making this project available for the many. 12 languages is more than I ever wished for, and I sincerely hope we can onboard even more in the future.

But if we want to make this work on the long run, and be able to onboard new translators for existing or new languages, we need to simplify the way it works right now. 

I suppose that the best way would be to move towards a dictionary based system. Basically, every single chunk of content should be mapped to a key, that would be shared between all translations. The Jekyll layer would get the correct content based on the current language and the key. This is how it works right now with the [“UI translations”](https://github.com/HugoGiraudel/sass-guidelines/blob/gh-pages/_locales/en.yml#L8-L25) in the `_locales` folder (Twitter button, donate partial, site baseline, etc.). We would need this for the whole thing, not only the UI. 

From there, there are a lot of options. We could either have a web interface to access these dictionaries (format to be defined: YAML, XLIFF, JSON, PO, etc.) such as [Pootle](http://pootle.translatehouse.org/). Or even a desktop application like [Poedit](http://poedit.net/), even if I think that makes it harder so probably not a good idea. There are more services allowing a similar workflow; I found [Zanata](http://zanata.org/), [POEditor](https://poeditor.com/) and [Transifex](https://www.transifex.com/) (although too expensive). I need to benchmark them a bit because I have no clue how they work.

Then translating some content or updating a translation would be as simple as accessing the dictionary of the relevant language (via a web UI for instance), change the content and save it. On my side, I will have to sync and deploy the translations on a regular basis, but that is really the easy part at this point.

## Any thought?

So here we are folks. I need your opinion on this.
What do you like in the current workflow?
What don’t you like?
What do you find annoying?
What would be ideal for you?
Do you have any hint?
Do you have any experience on translating content through a third-party service?
Please, speak your mind freely. I am **really** interested in anyone’s opinion.

Ping all: @pierrechoffe, @klapec, @matchabros, @morkro, @soniar4i, @davidkpiano, @grawl, @PavelDemyanenko, @airen, @pinggod, @kimdhoe, @AndersSchmidtHansen, @srescio, @HoBi, @ttomdewit, @bartveneman, @varemenos, @margaritis, @magalhini, @couto, @saravieira, @eduardoboucas, @jhonnymoreira, @shadeed, @nastiatikk.

**Comment by Grawl** [comment]
I have an experience with Poedit and gettext and found it nice to work with. But I have nothing to say about updates. Nor I translated a content once, nor a few updates, but not versions.

**Comment by lgiraudel** [comment]
In my opinion you should:
* extract translations in dedicated files with a key / value pair. In some formats, it's possible to use complex keys so it should be possible to use the english version as a key (but keep the possibility to overload the english version)
* extract translation files in a dedicated repository, used as a submodule: it will permit to send PR notifications to translators for changes in translations and not for changes in the source code. The better should be to trigger notifications only when the default version (english one) has some changes and not for every translation PR.

**Comment by martinschaible** [comment]
I worked recently on a translation project for a CMS (PresideCMS from the UK). The team used Zanata, which worked out very well. Very comfortable and easy to use. The output was managed by the project leader and the string ended in their repository as files with the extension ".properties." The problems i experienced are more on the human side:
- People translated text without using the software, ending having translated strings without a context.
- Not enough space in the UI: German strings are needing more space rather than english strings. French strings are consuming sometimes 50% more space rather than the english strings.
- Sometimes technical strings where, let's say "overtranslated". E.G. a "Spool folder" can be translated to "Spool-Ordner" for the german language.
- For a lot of CMS related translations, the world is using a kind of standardized strings. It might help to see how other products are looking in a foreign language.

In a other project (Pydio from France) i worked, we had to edit the i18n files directly. This worked out pretty good too. The benefit was, that a search and replace for some simple strings-types where possible, which saves time. The problem might be a merge in the repository, if several people are working on the same files. This can cost time for sure.

Independly, how the translation is organized, future updates can cause problems. E.g. The application gets new strings, which needs more translation.

I think, it is important, that the translated strings of an application are splitted to many smaller modules instead of having a big file.

**Comment by HugoGiraudel** [comment]
Hello y’all!

What is the current status with the update of your translation to 1.3? Shall I try to find some help for you on Twitter maybe? Please keep me updated. :)

Cheers!


@klapec, @matchabros, @soniar4i, @davidkpiano, @grawl, @PavelDemyanenko, @airen, @AndersSchmidtHansen, @srescio, @HoBi, @ttomdewit, @bartveneman, @varemenos, @margaritis, @magalhini, @couto, @saravieira, @eduardoboucas, @jhonnymoreira, @shadeed, @nastiatikk.

**Comment by HugoGiraudel** [comment]
I am super happy to introduce the Italian version (1.3) to the party: https://sass-guidelin.es/it/! :tada:

**Comment by douglasdeodato** [comment]
@HugoGiraudel  what is the story about the pt-br translate?

**Comment by HugoGiraudel** [comment]
I think the main problem right now is that it’s not up to date. :)

**Comment by douglasdeodato** [comment]
@HugoGiraudel  I can try Help on this, but pt is related to Portuguese from Portugal, pt-br is related to  Portuguese from Brazil, Off course is very similar but exist few differences when you write a phrase or word. I could not see pt-br , I will fork the project create a branch and start from scratch , I will try translare a session everyday, I will create a new Issue topic related for that if you don't mind. what do you think?  thank you.

**Comment by HugoGiraudel** [comment]
Sure. I’m fine with having a PT-BR translation if you feel like doing it. Keep me updated. :)

**Comment by jhonnymoreira** [comment]
@HugoGiraudel I've been through some stuff the last few months and kept myself away from everything related to programming. I got back this month and I saw @douglasdeodato interest on help. I'll try to set things up with him to deliver the most recent translation since I'm available full time right now [=

**Comment by douglasdeodato** [comment]
@jhonnymoreira  Hi Jhonny , off course we can start do this together why not 👍

**Comment by HugoGiraudel** [comment]
Hey y’all lovely contributors! I wish you a happy new year! ✨

Would you mind telling me if you plan on updating your translation (if not up-to-date) or if I should look up for some help from new translators? I’d like to have a rough idea of the team for the next few months. :)

Best!

**Comment by AndersSchmidtHansen** [comment]
Hi Hugo,

I won't be able to update the Danish translations for the time being due to
work. Sorry.

Hope you'll find someone to take over. :-)

Happy new year to you too!

Best,

*Anders Schmidt Hansen*
Co-Founder, Developer & Designer
*Pitcherific
<http://t.sidekickopen70.com/e1t/c/5/f18dQhb0SmZ58dDMPbW2n0x6l2B9nMJW7sM9dn7dK_MMdBzM2-04?t=https%3A%2F%2Fpitcherific.com%2F&si=5492243200999424&pi=c5756254-080b-4355-f8fe-da00a59a05c4>*
*Prepare Better Pitches. In Record Time.*

LinkedIn
<http://t.sidekickopen70.com/e1t/c/5/f18dQhb0SmZ58dDMPbW2n0x6l2B9nMJW7sM9dn7dK_MMdBzM2-04?t=https%3A%2F%2Fdk.linkedin.com%2Fin%2Fandersschmidthansen&si=5492243200999424&pi=c5756254-080b-4355-f8fe-da00a59a05c4>
 | Articles
<http://t.sidekickopen70.com/e1t/c/5/f18dQhb0SmZ58dDMPbW2n0x6l2B9nMJW7sM9dn7dK_MMdBzM2-04?t=http%3A%2F%2Fblog.pitcherific.com%2F&si=5492243200999424&pi=c5756254-080b-4355-f8fe-da00a59a05c4>
 | Twitter
<http://t.sidekickopen70.com/e1t/c/5/f18dQhb0SmZ58dDMPbW2n0x6l2B9nMJW7sM9dn7dK_MMdBzM2-04?t=https%3A%2F%2Ftwitter.com%2FTryPitcherific%2F&si=5492243200999424&pi=c5756254-080b-4355-f8fe-da00a59a05c4>


<http://t.sidekickopen70.com/e1t/c/5/f18dQhb0SmZ58dDMPbW2n0x6l2B9nMJW7sM9dn7dK_MMdBzM2-04?t=https%3A%2F%2Fwww.linkedin.com%2Fin%2Fandersschmidthansen&si=5492243200999424&pi=c5756254-080b-4355-f8fe-da00a59a05c4>

**Comment by morkro** [comment]
I will still maintain the German translation 🙋🏻

**Comment by soniar4i** [comment]
Hi everyone!

I hope I can update the spanish version soon :)

2017-01-16 12:06 GMT+01:00 Anders Schmidt Hansen <notifications@github.com>:

**Comment by HugoGiraudel** [comment]
Ping @klapec, @matchabros, @davidkpiano, @grawl, @PavelDemyanenko, @airen, @srescio, @HoBi, @ttomdewit, @bartveneman, @varemenos, @margaritis, @magalhini, @couto, @saravieira, @eduardoboucas, @jhonnymoreira, @shadeed, @nastiatikk.

(Sorry for the spam, I just would like to make sure everyone can be notified on my [last message](https://github.com/HugoGiraudel/sass-guidelines/issues/96#issuecomment-272832759). 😊)

**Comment by varemenos** [comment]
@margaritis and I will continue to work on the Greek translation.

**Comment by SaraVieira** [comment]
@HugoGiraudel I can take a look at PT-PT translation and see what is missing :)

**Comment by HugoGiraudel** [comment]
@SaraVieira The Portuguese translation is currently in 1.1 (English in 1.3). If you would like to sync it, I’d be happy to let you get started. :)

**Comment by SaraVieira** [comment]
@HugoGiraudel Already started by creating the branch and I am going commit by commit to see what is missing.
Should I add a pull request when I reach 1.2 or only on 1.3 ?

**Comment by HugoGiraudel** [comment]
It depends on how you want to do it. It might be easier to move from 1.1 to 1.2, then 1.2 to 1.3 and therefore to send 2 PRs in total.

Commits from 1.2: https://github.com/HugoGiraudel/sass-guidelines/blob/master/CHANGELOG.md#120-september-1st-2015.
Commits from 1.3: https://github.com/HugoGiraudel/sass-guidelines/blob/master/CHANGELOG.md#130-march-1st-2016

If you have any doubt or question, please ask. :)

**Comment by pierrechoffe** [comment]
@HugoGiraudel I will continue supporting the French translation.

**Comment by SaraVieira** [comment]
@HugoGiraudel can you please give me push access so I can send the modifications to the branch?

**Comment by HugoGiraudel** [comment]
@SaraVieira You could also fork the repository on your own GitHub account and create a PR from here? If you cannot do that for any reason, please ask and I’ll give you write access. :)

**Comment by pinggod** [comment]
@HugoGiraudel 

@airen and I will no longer support the Chinese translation(its latest version is 1.3).

**Comment by SaraVieira** [comment]
@HugoGiraudel Np, I forked the repository and I'm bumping it to 1.2

**Comment by HugoGiraudel** [comment]
@SaraVieira Let me know if I can be of any help.

@pinggod @airen Thank you for all you’ve done. :)

**Comment by HoBi** [comment]
@HugoGiraudel tomorrow I will have my last exam in a semester and then I wanna do a big update with translation.

**Comment by HugoGiraudel** [comment]
@HoBi Awesome to hear. Let me know if I can help. :)

**Comment by kimdhoe** [comment]
@HugoGiraudel I'll catch up slowly. 🏃

**Comment by HugoGiraudel** [comment]
@kimdhoe Terrific. If I can help in any way, please ask.

**Comment by JLammeer** [comment]
@HugoGiraudel Do you need help to update portuguese translation?

**Comment by HugoGiraudel** [comment]
@JLammeer Help is always welcome. The Portuguese translation is 2 versions behind (currently 1.1) so there is quite a bit of work to update it. If you feel like it though, I’ll be happy to merge a PR. :)

**Comment by HugoGiraudel** [comment]
The Portuguese version has been successfully updated to v1.3 thanks to @JLammeer’s hard work! Thank you so much. :) ✨

**Comment by JLammeer** [comment]
I'm happy to help.

**Comment by HugoGiraudel** [comment]
I’m very happy to announce that the Russian version is now up-to-date in 1.3 thanks to @arbaev fantastic work: https://sass-guidelin.es/ru/! ✨

**Comment by KittyGiraudel** [comment]
Hello friends,

I realized it would be nice to have the translation disclaimer in the current page language instead of always in English. So I was wondering if you could help me translate these 2 sentences.

> You are viewing the {language} translation from {translator A} and {translator B} of the original Sass Guidelines from Kitty Giraudel.
>
> This version is exclusively maintained by contributors without the review of the main author, therefore might not be completely authentic.

Note:
- `{language}` is the name of your language.
- I will replace `{translator A}` and `{translator B}` (and more when needed) with your names. 😅 
- I will deal with the implementation, I just need your help with translations. :)

Thank you! 💖

**Comment by noah-vdv** [comment]
U bekijkt de nederlandse vertaling van [Noah van der Veer](https://github.com/noah-vdv) van de originele [Sass-richtlijnen](https://sass-guidelin.es/) van [Kitty Giraudel](https://kittygiraudel.com/).

Deze versie wordt exclusief onderhouden door bijdragers zonder de beoordeling van de hoofdauteur en is daarom mogelijk niet volledig authentiek.

**Comment by morkro** [comment]
Thank you @KittyGiraudel! Here's the German version:

> Du schaust dir gerade die deutsche Übersetzung von Moritz Kröger der originalen Sass Guidelines von Kitty Giraudel an.
>
> Diese Version wird exklusiv durch die Übersetzer gepflegt, ohne zusätzliche Überprüfung des Hauptautors, und mag deshalb nicht vollständig authentisch sein.

**Comment by varemenos** [comment]
Here is the Greek version, courtesy of @modkaffes

```
Βλέπετε την ελληνική μετάφραση των Μόδεστο Καφφέ, Σεμπαστιέν Νικολάου, Άδωνη Κακουλίδη και Κωνσταντίνου Μαργαρίτη για τις πρωτότυπες Οδηγίες για Sass των Kitty Giraudel.
Αυτή η έκδοση συντηρείται αποκλειστικά από συνεισφέροντες χωρίς τον έλεγχο του κύριου συντάκτη, επομένως μπορεί να μην είναι εντελώς αυθεντική.
```

@KittyGiraudel could you please include @modkaffes and @sebdesign in the list of Greek translators? They are included in the website but not in this issue (which is most likely my fault 😞).

**Comment by Afsaneh1361** [comment]
> 💢 **به روزرسانی مارس '21: برای حفظ ترجمه های لهستانی ، چکی و دانمارکی کمک می خواهید.**
> 
> بگذارید این موضوع را باز کنم تا بتوانم با همه شما عزیزان که موافق ترجمه یکباره دستورالعمل ها هستند صحبت کنم. اگر هرکدام از شما در مورد چیزی س anythingال دارند ، لطفاً آزادانه بپرسید!
> 
> ## شروع شدن
> 1. برای کسب اطلاعات بیشتر در مورد Git و سیستم انشعاب مورد استفاده ، [CONTRIBUTING.md را](https://github.com/HugoGiraudel/sass-guidelines/tree/master/CONTRIBUTING.md) بخوانید .
> 2. برای کسب اطلاعات بیشتر در مورد نحوه ترجمه واقعی دستورالعمل ها ، [TRANSLATING.md را](https://github.com/HugoGiraudel/sass-guidelines/tree/master/TRANSLATING.md) بخوانید .
> 
> _توجه: اگر به جای شروع کار از ابتدا ترجمه را به روز می کنید ، با [CHANGELOG.md](https://github.com/HugoGiraudel/sass-guidelines/tree/master/CHANGELOG.md) به خود کمک کنید ._
> 
> ## ارسال کار خود
> 1. مطمئن شوید نسخه شما با `master`شعبه فعلی به روز است .
> 2. ارسال [درخواست کشش](https://github.com/hugogiraudel/sass-guidelines/pulls) در برابر `master`شاخه بر روی GitHub.
> 3. دیگه تموم شد
> 
> ## ترجمه ها
> وضعیت	زبان	نسخه	مترجمان
> ✅	چینی ها	1.3	کمک خواست
> ✅	آلمانی	1.3	morkro
> ✅	فرانسوی	1.3	pierrechoffe
> ✅	ایتالیایی	1.3	MakhBeth ، mattcav
> ✅	یونانی	1.3	varemenos ، margaritis
> ✅	اسپانیایی	1.3	@ soniar4i ، davidkpiano
> ✅	پرتغالی	1.3	@magalhini, @Couto, @SaraVieira, @eduardoboucas, @hunkjazz
> ✅	Russian	1.3	@irene-97, @PavelDemyanenko, @nastiatikk, @Afsaneh

**Comment by MakhBeth** [comment]
Stai leggendo la traduzione italiana di [Davide Di Pumpo](https://github.com/MakhBeth) e [Matteo Cavucci](https://github.com/mattcav) delle originali linee guida per Sass di Kitty Giraudel.

Questa versione è mantenuta esclusivamente dai collaboratori, senza la revisione dell'autore principale. Per questo potrebbe non essere completamente autentica.
